lib/ops/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ops
-  VERSION = '1.1.1'.freeze
+  VERSION = '1.1.2'.freeze
```

Comment:
Shouldn't this be version `1.2.0` since this bumps the bundler and Ruby versions?

Reply:
good point Trevor 👍

Reply:
@rkledesma let's update to suggested version and try to merge it today